yaki_backend/src/declaration.router.ts

```diff
@@ -63,5 +63,73 @@ declarationRouter.get(
   }
 );
 
+declarationRouter.get(
+  "/declarations/declaredDays",
+  (req, res, next) => {
+    if (req.query.userId == null) {
+      return res.status(400).send({message: "Missing required query parameter 'userId'"})
+    }
+    if (isNaN(parseInt(req.query.userId as string))) {
+      return res.status(400).send({message: "Invalid userId"})
+    }
+    return authService.verifyToken(req, res, next)
+  },
+  async (req, res) => {
+  /**
+   * @swagger
+   * declarations/declaredDays:
+   *   get:
+   *     summary: Get declared days by user ID
+   *     description: Returns a list of declared days for a specific user.
+   *     tags:
+   *       - Declarations
+   *     parameters:
+   *       - in: query
+   *         name: userId
+   *         description: The ID of the user to retrieve declared days for.
+   *         required: true
+   *         schema:
+   *           type: integer
+   *           minimum: 1
+   *     responses:
+   *       200:
+   *         description: OK. Returns a list of declared days for the user. Returns [] if the user doesn't exist.
+   *         content:
+   *           application/json:
+   *             schema:
+   *               type: array
+   *               items:
+   *                 $ref: '#/components/schemas/DeclaredDay'
+   *       400:
+   *         description: Bad Request. Missing required query parameter 'userId'.
+   *         content:
+   *           application/json:
+   *             schema:
+   *               type: object
+   *               properties:
+   *                 message:
+   *                   type: string
+   *                   description: The error message.
+   *       500:
+   *         description: Internal Server Error.
+   *         content:
+   *           application/json:
+   *             schema:
+   *               type: object
+   *               properties:
+   *                 message:
+   *                   type: string
+   *                   description: The error message.
+   */
```

Comment:
If those comments is not useful or information about code, its better to delete it

Reply:
Swagger documentation. I don't know the project so I don't know if they are used to generate the Swagger. Do you know ?